internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -46,16 +46,24 @@ const (
 	GRPCServerUseTLSEnvVar = "GRPC_SERVER_USE_TLS"
 	// GRPCServerTLSCertEnvVar is the grpc server tls cert.
 	GRPCServerTLSCertEnvVar = "GRPC_SERVER_TLS_CERT"
-	// GRPCServerTLSClientCertFilename is the GRPC tls cert file.
-	GRPCServerTLSCertFilename = "/certs/tls.crt"
+	// GRPCTLSCertFilename is the GRPC tls cert file.
+	GRPCTLSCertFilename = "/certs/tls.crt"
```

Comment:
prefer to

Reply:
I did this because this cert not only be used by GRPC Service between rate limit services and envoy but also in rate limited service and gateway.

Reply:
then move thems to `Kubernetes/resource/resource.go`. The  commom resources and constants have defined here

Reply:
IMO, maybe needs different certs.

Reply:
https://github.com/envoyproxy/gateway/pull/1499#discussion_r1220097937 
I think we do not need to add another TLS cert, client and server cert are the same in another way

Reply:
two parts needs certs
- EG to ratelimit
- ratelimit to redis.server

I don't think we should use the same certs. Because the redis backend is not standard, the users can design it by themselves. 

cc @arkodg

Reply:
@qicz there are 3 connections in play
1) ratelimit to xds server/EG
2) ratelimit to envoy
3) ratelimit to redis

this PR only solves 1, and has solved 2 already, 3 will always be user input